src/components/twilio-voice-dialer/public/twilio-voice-dialer.js

```diff
@@ -33,15 +56,39 @@ class TwilioVoiceDialer extends HTMLElement {
             placeholder="recipient"
             id="recipient"
             value=${this.getAttribute('recipient')}>
+				<button id="call">Call</button>
+    		<button id="hangup">Hangup</button>
```

Comment:
Nit: indentation.

That brings up a good question actually, do we want a formatter/linter like `prettier`? Or is that too opinionated?

Reply:
Not sure `prettier` would have even caught this one to be honest, since it's just a templated string.

Reply:
Hmmm identation is a little wonky here. I aligned, but still very indented, any ideas?

This has `prettier` formatting, but not sure if we want to enforce it since our other repos do not.